test/cypress/performance/minimal.cy.js

```diff
@@ -18,11 +18,11 @@ const desktopConfig = {
 };
 
 describe('Lighthouse tests', () => {
-    it('Climate demo', () => {
-        cy.visit('/dashboards/demo/climate');
+    it('Minimal demo', () => {
+        cy.visit('/dashboards/demo/minimal');
         cy.lighthouse(
             {
-                performance: 42
+                performance: 0
```

Comment:
Is this correct?

Reply:
Kind of, seems you have to set something other than `undefined` there to only run the performance tests. (Could not provide the whole object, but then other categories are also run).

The test will fail [if this comparison](https://github.com/highcharts/highcharts/pull/19590/files#diff-cf22095590be315f44cba8f347076fdf3036fb674ed60d85422d762f7085d9c0R50) is not met, so this threshold is not really needed.